Funcky.Test/Extensions/EnumerableExtensions/TransposeTest.cs

```diff
@@ -8,6 +8,23 @@ namespace Funcky.Test.Extensions.EnumerableExtensions
 {
     public sealed class TransposeTest
     {
+        [Fact]
+        [SuppressMessage("ReSharper", "ReturnValueOfPureMethodIsNotUsed", Justification = "The ToList just enumerates the items, we test for the side effect in the next line.")]
```

Comment:
Couldn't you suppress this warning by explicitly discarding the expression (`_ = expr`)?

Reply:
I wasnt aware that I can do that in a lambda, adressed with 1d36f76ed0506af317e440696cbcd1f3d0602739